app/assets/javascripts/initializers/initializeBodyData.js

```diff
@@ -20,15 +27,15 @@ function fetchBaseData() {
       meta.name = 'csrf-param';
       meta.content = json.param;
       document.getElementsByTagName('head')[0].appendChild(meta);
-      var meta = document.createElement('meta');
+      var meta = document.createElement('meta'); // eslint-disable-line no-redeclare
```

Comment:
what about using a different variable instead of disabling this check?

Reply:
Yep, that was my first thought, but since this is my first ever contribution to someone else’s code base, I was super nervous about breaking things. I’ll make that change though.

Reply:
ahah I can understand! :-) If you don't feel comfortable let me know, I think the PR is already strong